routes/web.php

```diff
@@ -52,7 +56,29 @@
     return view('planner.edit');
 });
 
+Route::get('/friends/add', function () {
+    return view('friends.add');
+});
+
+Route::get('/friends/edit', function () {
+    return view('friends.edit');
+});
```

Comment:
i would've suggested resource controller here, but if I saw it right, you've already started to implement them